src/categories.ts

```diff
@@ -19,3 +19,5 @@ const UNKNOWN = 'Unknown';
 const categories = [UNKNOWN, INTELLIGENCE, CONTROLLER, SENSOR, SIDEKICK];
 
 export { CONTROLLER, INTELLIGENCE, SENSOR, SIDEKICK, UNKNOWN, categories };
+
+export default { CONTROLLER, INTELLIGENCE, SENSOR, SIDEKICK, UNKNOWN, categories };
```

Comment:
Since we're enuming it, we should enum all the way (sorry, I think I said I hadn't gotten around to doing it for Categories implying that I'd take care of it myself but since we're creating other enums here it's probably best to do this in the same PR):
```suggestion
export default enum Category {
  CONTROLLER = 'controller',
  INTELLIGENCE = 'Intelligence',
  SENSOR = 'Sensor',
  SIDEKICK = 'Sidekick',
  UNKNOWN = 'Unknown',
}
```

Reply:
This doesn'a allow for the export of the `categories` array. How would you arrange this so TS doesn't blow up and we have access to these values in array form?

Reply:
Typescript String enums (as opposed to their numeric enums which operate somewhat differently) are converted into a POJO at runtime:
```typescript
enum Bob {
    Hello = "bob",
    Bye = "bob2"
}
```
transpiles to:
```javascript
var Bob;
(function (Bob) {
    Bob["Hello"] = "bob";
    Bob["Bye"] = "bob2";
})(Bob || (Bob = {}));
```
So we can get the values with `Object.values(Category)`. This is the sort of thing that I'd normally lean towards letting the user parse through, however not all of them might be using typescript and for their ease of use we could do something like:

```
export default enum Category { ... }
export const categories = Object.values(Category);
```
And then people can consume that by importing `import { categories } from './categories';`